src/test/java/jenkins/plugins/slack/workflow/SlackSendStepTest.java

```diff
@@ -0,0 +1,78 @@
+package jenkins.plugins.slack.workflow;
+
+
+import hudson.model.Result;
+import jenkins.plugins.slack.Messages;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.jenkinsci.plugins.workflow.steps.StepConfigTester;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runners.model.Statement;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+public class SlackSendStepTest {
+    @Rule
+    public RestartableJenkinsRule story = new RestartableJenkinsRule();
```

Comment:
You are using only one step per test, it's simpler to use `JenkinsRule` instead of `RestartableJenkinsRule`.